.changeset/late-countries-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@eth-optimism/contracts': patch
+---
+
+Update contracts README to add deploy instructions.
```

Comment:
I think this changeset is unnecessary since no code changes have occurred.

Reply:
I think we should leave it - it will populate the NPM package's website if we do a release with an updated `README`

Reply:
hm, ok